content/en/docs/workshop/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Contributor Workshop
```

Comment:
We should not include any markdown files in the content directory because it will come up when published.
https://deploy-preview-220--kubernetes-contributor.netlify.app/docs/workshop/contributing/